setup.py

```diff
@@ -348,7 +349,8 @@ def main():
             "include/nvfuser/ir/*.h",
             "include/nvfuser/python_frontend/*.h",
             "include/nvfuser/scheduler/*.h",
-            "include/nvfuser/serde*.h",
+            "include/nvfuser/serde/*.h",
+            "include/nvfuser/flatbuffers/*.h",
```

Comment:
imagine that once we have dynamic_type header in separate folder, we'll also want to package that?!